src/http_extension.ts

```diff
@@ -262,11 +254,11 @@ export class HttpExtension implements IHttpExtension {
     return routerNames;
   }
 
-  protected onStarted(): Promise<any> | any {return; }
+  protected onStarted(): Promise<any> | any { return; }
 
   protected initializeBaseMiddleware(app: Express.Application): void {
 
-    const options: {[optionName: string]: any} = {};
+    const options: { [optionName: string]: any } = {};
```

Comment:
Previous version was better.
```suggestion
    const options: {[optionName: string]: any} = {};
```